Peach Fuzzer small fixes

katlapinka opened the pull request:
- fix for download and copy peach fuzzer files
- removed fuzzed command prefix/suffix and encoding

Needs: https://github.com/Open-CAS/test-framework/pull/6

pep8speaks commented:
Hello @KamilLepek! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:







* In the file [`test_package/conftest.py`](https://github.com/Open-CAS/test-framework/blob/45b677b5be87fc111e59fbc3d0933eac0c8b931f/test_package/conftest.py):

> [Line 52:23](https://github.com/Open-CAS/test-framework/blob/45b677b5be87fc111e59fbc3d0933eac0c8b931f/test_package/conftest.py#L52): [E126](https://duckduckgo.com/?q=pep8%20E126) continuation line over-indented for hanging indent

Commit:
Peach Fuzzer small fixes

- fix for download and copy peach fuzzer files
- removed fuzzed command prefix/suffix and encoding

Signed-off-by: Katarzyna Treder <katarzyna.treder@h-partners.com>

Deixx commented on test_tools/peach_fuzzer/peach_fuzzer.py:
```diff
@@ -28,7 +29,7 @@ class PeachFuzzer:
     base_dir = "/root/Fuzzy"
     peach_dir = "peach-3.0.202-linux-x86_64-release"
-    xml_config_template = posixpath.join(posixpath.dirname(__file__), "config_template.xml")
+    xml_config_template = posixpath.join(os.path.dirname(__file__), "config_template.xml")
```
is this intended to mix os.path and posixpath here? If it's on the executor side then it should be os.path all the way - including line 119 (and 134?)

pep8speaks commented:
Hello @katlapinka! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:

* In the file [`test_tools/peach_fuzzer/peach_fuzzer.py`](https://github.com/Open-CAS/test-framework/blob/3b875165d9c16b18ee8e42ac7e56224c2b412786/test_tools/peach_fuzzer/peach_fuzzer.py):

> [Line 90:3](https://github.com/Open-CAS/test-framework/blob/3b875165d9c16b18ee8e42ac7e56224c2b412786/test_tools/peach_fuzzer/peach_fuzzer.py#L90): [E111](https://duckduckgo.com/?q=pep8%20E111) indentation is not a multiple of four
> [Line 134:7](https://github.com/Open-CAS/test-framework/blob/3b875165d9c16b18ee8e42ac7e56224c2b412786/test_tools/peach_fuzzer/peach_fuzzer.py#L134): [E111](https://duckduckgo.com/?q=pep8%20E111) indentation is not a multiple of four
> [Line 159:3](https://github.com/Open-CAS/test-framework/blob/3b875165d9c16b18ee8e42ac7e56224c2b412786/test_tools/peach_fuzzer/peach_fuzzer.py#L159): [E111](https://duckduckgo.com/?q=pep8%20E111) indentation is not a multiple of four
> [Line 159:3](https://github.com/Open-CAS/test-framework/blob/3b875165d9c16b18ee8e42ac7e56224c2b412786/test_tools/peach_fuzzer/peach_fuzzer.py#L159): [E301](https://duckduckgo.com/?q=pep8%20E301) expected 1 blank line, found 0
